internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -574,3 +574,38 @@ func convertCommand(command *manifest.CommandOverride) ([]string, error) {
 	}
 	return out, nil
 }
+
+func convertPublish(p *manifest.PublishConfig) (*template.PublishOpts, error) {
+	if p == nil || p.Topics == nil {
```

Comment:
in case `p.Topics` is `[]string{}`
```suggestion
	if p == nil || len(p.Topics) == 0 {
```
even if `p.Topics` is `nil` i think checking the length is still robust enough.